src/status_im/ui/components/badge.cljs

```diff
@@ -10,4 +11,11 @@
                {:background-color colors/blue
                 :justify-content :center
                 :align-items :center})
-   [react/text {:style {:typography :caption :color colors/white}} label]])
\ No newline at end of file
+   [react/text {:style {:typography :caption :color colors/white}} label]])
+
+(defn message-counter
+    [value & [small?]]
+    (let [more-than-99 (> value 99)]
```

Comment:
There's no real need in this `let` form as `more-than-99` is used only once and thus the condition can be inlined in `if`.